lib/core/versions/latest/RequestHandler.ts

```diff
@@ -27,7 +30,7 @@ export default class RequestHandler implements IRequestHandler {
     console.info(`Handling operation request of size ${request.length} bytes...`);
 
     // Perform common validation for any write request and parse it into an `Operation`.
-    let operation: Operation;
+    let operation: Operation | OperationModel;
```

Comment:
Shouldn't parse always return type `OperationModel` on line `operation = await Operation.parse(request);`, so why does this need to be type Operation | OperationModel?

Reply:
You are right, it was at one point in development incompatible, but it is now! Updated.